concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+
+### Sprint Planning
+With our backlog and sprint boards established, we can flesh out our first meeting: sprint planning.
+
+This is meant to be a time to consider what work the team wants to accomplish in the next two-week sprint and assign relevant tasks to people. It should happen on Mondays and take about an hour, following the sprint retrospective (detailed later).
+
+It should start with a brief discussion about the team’s goals. This should be an open, but high-level, discussion that will help frame the kinds of issues we should prioritize. It shouldn’t take long -- no more than 15-20 minutes.
+
+With the context of that discussion in mind, we’ll move to backlog review. As detailed above, our backlogs are contained in individual boards, and we’ll run through each one as part of the planning process.
+
+For each board:
+- Briefly run through the Backlog column to decide if there are any issues we want to promote to "Tracking" status.
+- Run through all items in "Tracking" and decide which ones are highest priority, and who will work on them. They will become the issue’s assignee.
+
+From there, automation will aid the process -- any newly assigned issues will:
+- be moved to the Active column in the backlog board
+- automatically added to the Assigned column of the Active Sprint board
+
+This process repeats for every board, after which our Active Sprint board will be populated with our workload for the next two weeks, and our sprint planning is complete.
+
+### Daily Standup
+Our daily standups will take place every day we don’t have a longer meeting (such as sprint planning/retro) for half an hour. This is meant to be a quick check-in to ensure newly-opened issues get quick responses and blockers get resolved.
+
+First, if there are any new people on the call, we'll welcome them and allow them to ask questions or have discussions as needed.
+
+Next, we’ll have a look at all untriaged issues -- any that do not have a label, and are not part of a project. This will allow the whole team to discuss them and ensure they get moved to the appropriate backlog board and labeled correctly. This shouldn’t take more than 15 minutes, though that’s a soft limit -- if a good conversation is happening, it shouldn’t be cut off.
+
+Finally, if there's time, we'll dedicate some time to a quick review of our Current Sprint board. This will serve as an opportunity for developers to elaborate on any blockers and ask for reviews or re-reviews as needed. We should strive to do this every day if possible, but if prior discussion makes that infeasible, we don't _have_ to do it (there's always tomorrow). Project discussion is more important.
+
+### Sprint Retrospective
+Our final meeting is the sprint retrospective. This should be a quick half-hour meeting that takes the place of Monday’s standup where we discuss:
```

Comment:
I think we should err on the side of caution and block off 60 minutes for the retrospective.

Reply:
A 60 minute retrospective could make sense. If we do this and the planning meeting is the same day, I would strongly vote to have them back-to-back so that if the retrospective only takes 25 minutes we can immediately transition into planning, instead of having an awkward gap between the two and having two hours blocked off that we don't really need but where it's hard to get work done.

Reply:
I agree we need more time. I'd also like planning and retrospective to be back-to-back.

Reply:
Done in latest version. Resolving to simplify open feedback.